package.json

```diff
@@ -11,37 +11,37 @@
     "@babel/core": "^7.9.0",
     "@babel/preset-env": "^7.9.5",
     "@frctl/fractal": "^1.2.1",
-    "@netzstrategen/fractal-twig": "^2.2.4",
+    "@netzstrategen/fractal-twig": "^2.3.1",
```

Comment:
Why don't we update to the most recent release 2.4.0? 🤔

Reply:
Probably an oversight, i will correct that. Though it should not matter, because the "^" only limits the first digit IIRC.